audformat/core/utils.py

```diff
@@ -326,6 +326,36 @@ def expand_file_path(
     return index
 
 
+def has_overlap(
+        obj: typing.Union[pd.Index, pd.DataFrame, pd.Series],
+) -> bool:
+    r"""Check if one or more segments overlap.
```

Comment:
```suggestion
    r"""Check if one or more segments in the index overlap.
```

Since we now support `Series` and `DataFrame` I think it's better to be more precise here.